Course/Course/Presentation/Dates/CourseDatesView.swift

```diff
@@ -41,7 +41,54 @@ public struct CourseDatesView: View {
             }
             
             if viewModel.dueDatesShifted {
-                DatesShiftedSuccessView(selectedTab: .dates, courseDatesViewModel: viewModel)
+                DatesSuccessView(
+                    title: CourseLocalization.CourseDates.toastSuccessTitle,
+                    message: CourseLocalization.CourseDates.toastSuccessMessage,
+                    selectedTab: .dates,
+                    courseDatesViewModel: viewModel
+                )
+            }
+            
+            if viewModel.calendarEventsAdded {
+                DatesSuccessView(
+                    title: CourseLocalization.CourseDates.calendarEvents,
+                    message: CourseLocalization.CourseDates.calendarEventsAdded,
+                    selectedTab: .dates
+                ) {
+                    Task {
+                        await MainActor.run {
+                            viewModel.calendarEventsAdded = false
+                        }
+                    }
+                }
```

Comment:
This code is being shared between the following three conditions `calendarEventsAdded`, `calendarEventsRemoved`, and `calendarEventsUpdated`. How about moving it to a function and use that. Even that func can be used for `dueDatesShifted` with an additional param.